extensions/3DTILES_binary_buffers/README.md

```diff
@@ -0,0 +1,164 @@
+# 3DTILES_binary_buffers
+
+## Overview
+
+This extension to 3D Tiles enables storage of binary data in external buffers.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_binary_buffers.bufferViews
+
+`bufferViews` provide a typed view into a `buffer`.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**buffer**|`integer`|The index of the buffer.|☑️ Yes|
+|**byteOffset**|`integer`|The offset relative to the start of the buffer in bytes.|☑️ Yes|
+|**byteLength**|`integer`|The length of the bufferView in bytes.| ☑️ Yes|
+|**elementCount**|`integer`|The number of elements in the buffer view.| ☑️ Yes|
+|**elementByteOffsetsBufferView**|`integer`|The index of the bufferView containing byte offsets for each element. Must be defined for the STRING type.|No|
```

Comment:
Can `elementByteOffsetBufferView` be used for properties containing arbitrary sized chunks of binary data? What would be the `elementType` in that case?

Reply:
Sorry, duplicate comment to https://github.com/CesiumGS/3d-tiles/pull/415#discussion_r437015903. I thought github lost this one.

Reply:
Yes, a `NONUNIFORM` element type will be introduced to meet the requirements for `EXT_3dtiles_feature_metadata`.